docs/source/cli/xcube_serve.rst

```diff
@@ -471,11 +521,78 @@ Example
     [I 190924 17:08:54 service:159] press CTRL+C to stop service
 
 
+Server Demo Configuration File for DataStores
+=============================================
+The server configuration file consists of various parts, some of them are necessary, others are optional.
+Here the `demo stores configuration file`_ used in the `example stores`_ is explained in detail.
+
+This configuration file differs only in one part compared from `config`_. This part is `data stores`_.
```

Comment:
```suggestion
This configuration file differs only in one part from :ref:`config`.
`the configuration above`_: `data stores`_.
```

Most of the configuration of locally stored datasets is equal to the configuration of
:ref:`remotely-stored-xcube-datasets`.
`remotely stored xcube datasets`_.